src/components/Booklist.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Book from './Book';
+
+function Booklist() {
+  const books = [
+    { title: 'Book1', author: 'Author1' },
+    {
+      title: 'Book2',
+      author: 'Author2',
+    },
+    {
+      title: 'Book3',
+      author: 'Author3',
+    },
+    {
+      title: 'Book4',
+      author: 'Author4',
+    },
+    {
+      title: 'Book5',
+      author: 'Author5',
+    },
+  ];
+
+  // return each book inside a <ul> tag and a remove button for each book
+  return (
```

Comment:
- [ ] OPTIONAL Please, consider removing comments from your code, to make it clear - your code in general very clear and understandable and do not need any comments for instruction. 👍